lib/resources/dstu2/condition.yaml

```diff
@@ -101,10 +101,13 @@ fields:
     terminology:
     - display: SNOMED CT
       system: http://snomed.info/sct
```

Comment:
The system shows up in brackets. Seems a bit off. Can we do like a little bullet below the display with "system: http://snomed.info/sct"? That would be more clear to indicate, it is a system to use in codings?